web/packages/teleterm/src/ui/TopBar/AdditionalActions.tsx

```diff
@@ -62,6 +62,29 @@ function useMenuItems(): MenuItem[] {
   const isDarwin = platform === 'darwin';
 
   const menuItems: MenuItem[] = [
+    {
+      title: 'Create Connect My Computer role',
```

Comment:
As mentioned in the commit message, this is temporary and will be removed once #28891 is merged. I just wanted to give the reviewers of that PR an easy way to actually test the endpoint.